main/guides/getting-started/README.md

```diff
@@ -11,6 +14,12 @@ Before getting started, there are some resources you might want to keep handy in
 - Send us a message on [X](https://twitter.com/agoric)
 - Send an Email to [Developer Relations](mailto://devrel@agoric.com)
 
+::: tip Video Walkthrough
+
+As you're going through this tutorial it may be helpful to watch this video walkthrough, showing you all the steps and the desired outcome of each.
+
+<iframe width="560" height="315" src="https://youtu.be/huTI9_Gq2ow" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
this isn't working for me in the deploy preview: "www.youtube.com refused to connect"

![image](https://github.com/Agoric/documentation/assets/150986/28e7af53-572f-4a71-850c-6412319f2cf7)

Reply:
Updated the YouTube embed URL, should be loading now...